Tests/Remora.Discord.Rest.Tests/API/Webhooks/DiscordRestWebhookAPITests.cs

```diff
@@ -817,6 +817,7 @@ public async Task PerformsInThreadRequestCorrectly()
                 b => b
                     .Expect(HttpMethod.Post, $"{Constants.BaseURL}webhooks/{webhookId}/{token}")
                     .WithQueryString("wait", shouldWait.ToString())
+                    .WithQueryString("thread_id", threadID.ToString())
```

Comment:
Can you double-check that both parameters are detected with a query string check like this?

Reply:
Yeah, having an exact matcher would be good - can you update this PR and open an issue?

Reply:
Ok, pushed that change for this code here and submitted the issue as #309